test/messages-test.js

```diff
@@ -1,40 +1,48 @@
+import Gists from 'gists'
+import GoogleSpreadsheet from 'google-spreadsheet'
 import async from 'async'
 import chai from 'chai'
 import sinon from 'sinon'
 let messages = require('../scripts/messages.js').messages
 
 
 describe('Google Docs and Gists Logging', function () {
-    it('Google Docs messaging', function () {
-        let message = {
-            "text": "+todo testing @test",
-            "user": {
-                "name": "test"
-            }
-        }
-        sinon.stub(messages, "sendMessage", function () {
-            return "0"
+    let save
+
+    beforeEach(function () {
+        save = null
+    })
+
+    it('Formatting message', function () {
+        save = sinon.stub(messages, 'getRoom').callsFake(function (roomId, callback) {
+            sinon.assert.calledOnce(save)
```

Comment:
Can we also test that this function was call with correct/expected parameters?
```
{
  "action": "expected",
  "timestamp": "expected",
  "poster": "expected",
  "assignees": "expected",
  "message": "expected",
  "room": "expected"
}
```
Eg: 
- test against expected output
- test when +todo is missing it should not be called at all
- test when it is in the middle